NEWS.md

```diff
@@ -84,6 +135,23 @@ window.inactive.button.shade.unpressed.image.color
 
 ### Fixed
 
+- Fix button release events sometimes not being sent (#2226)
+- Fix xdg-shell popups on wrong output with some Qt themes. (#2224)
+- Take into account xdg-shell minimum window size for resizing. This is
+  relevant when using `<resize drawContents="no">` (#2221)
+- Fix button scaling issues (#2207, #2225)
+- Add portals.conf file, amend `labwc.desktop` and modify `XDG_CURRENT_DESKTOP`
```

Comment:
That is a very long sentence to digest, may be splitting into several sentences.